internal/policy/operator/deployable_by_olm.go

```diff
@@ -362,7 +370,7 @@ func (p *DeployableByOlmCheck) generateOperatorGroupData(operatorData *operatorD
 	case operatorsv1alpha1.InstallModeTypeAllNamespaces:
 		targetNamespaces = []string{}
 	}
-	log.L().Debugf("The OperatorGroup's TargetNamespaces is %s", targetNamespaces)
+	logger.V(1).Info("OperatorGroup TargetNamespaces", "namespace", targetNamespaces)
```

Comment:
There are a couple of spots where we lose the phrasing of the log line in order to utilize the k/v structure. I'm not sure how I feel about it, so I'm just putting this here as a reminder to see if it seems off to anyone else.

Reply:
I get that. For this particular one, I don't think that's the case. But if you do find others that are candidates, point them out.